tests/generators/test_classic.py

```diff
@@ -1,6 +1,21 @@
+import pytest
 import xgi
 
 
 def test_empty_hypergraph():
     H = xgi.empty_hypergraph()
     assert (H.num_nodes, H.num_edges) == (0, 0)
+
+def test_star_clique(): 
+
+    with pytest.raises(ValueError):
+        H = xgi.star_clique(-1, 7, 3)
+    with pytest.raises(ValueError):
+        H = xgi.star_clique(6, -1, 3)
+    with pytest.raises(ValueError):
+        H = xgi.star_clique(6, 7, -1)
+    with pytest.raises(ValueError):
+        H = xgi.star_clique(6, 7, 7)
+
+    H = xgi.star_clique(6, 7, 3)
+    assert H.num_nodes == 13
```

Comment:
Suggestion: this test seems a bit weak - can you please add a few more asserts? Like for the number of simplices, maximal order, etc.

Reply:
Done